geometry/geometry_config.py

```diff
@@ -116,7 +116,7 @@
     magnetIncrease    = 100.*u.cm
     # make z coordinates for the decay volume and tracking stations relative to T4z
     # eventually, the only parameter which needs to be changed when the active shielding lenght changes.
-    z4=2438.*u.cm+magnetIncrease+extraVesselLength
+    z4 = 2582.75*u.cm + magnetIncrease + extraVesselLength
```

Comment:
Could you make 2582.75 cm a named constant? It would help make the logic clearer if we knew what that number means.

```suggestion
    z4 = 2582.75 * u.cm + magnetIncrease + extraVesselLength
```